__tests__/app.test.js

```diff
@@ -1,62 +1,112 @@
 require('dotenv').config();
 
 const { execSync } = require('child_process');
-
 const fakeRequest = require('supertest');
 const app = require('../lib/app');
 const client = require('../lib/client');
+// const animalData = require('../data/animals.js');
 
 describe('app routes', () => {
   describe('routes', () => {
-    let token;
+    // let token;
   
     beforeAll(async () => {
       execSync('npm run setup-db');
   
       await client.connect();
-      const signInData = await fakeRequest(app)
-        .post('/auth/signup')
-        .send({
-          email: 'jon@user.com',
-          password: '1234'
-        });
+      // const signInData = await fakeRequest(app)
+      //   .post('/auth/signup')
+      //   .send({
+      //     email: 'jon@user.com',
+      //     password: '1234'
+      //   });
       
-      token = signInData.body.token; // eslint-disable-line
-    }, 10000);
+      // token = signInData.body.token; // eslint-disable-line
+    }, 100000);
   
     afterAll(done => {
       return client.end(done);
     });
 
-    test.skip('returns animals', async() => {
-
-      const expectation = [
-        {
-          'id': 1,
-          'name': 'bessie',
-          'cool_factor': 3,
-          'owner_id': 1
-        },
-        {
-          'id': 2,
-          'name': 'jumpy',
-          'cool_factor': 4,
-          'owner_id': 1
-        },
-        {
-          'id': 3,
-          'name': 'spot',
-          'cool_factor': 10,
-          'owner_id': 1
-        }
-      ];
+    test('POST /create a new animal in animal data', async ()=>{
+      const newAnimal = {
+        id:5,
```

Comment:
Keep in mind leaving this id in the data means that whenever a user creates a new animal to add to your list, they have to provide the id themselves and won't know if the app already has that id already. It works, but it's bad user experience.